airflow/providers/microsoft/azure/hooks/asb_admin_client.py

```diff
@@ -0,0 +1,83 @@
+# Licensed to the Apache Software Foundation (ASF) under one
```

Comment:
I think we should have unified `hooks/asb.py` with classes `AdminClientHook`, `MessageHook`
That is also how we do it for the other

@josh-fell WDYT?

Reply:
In intial development stage `BaseAzureServiceBusHook` ,`AzureServiceBusAdminClientHook`  and `ServiceBusMessageHook` was in same file after seeing the `AzureBaseHook` and other azure class I moved to different file :) . I am fine with your suggestion lets rename the class names and keep it under same file, lets wait for @josh-fell reply as well

Reply:
forgot to add one more reason for keeping it in separate file is there is some more operators is on the way I have added Azure Service bus subscription based operators and hooks changes as draft PR and Azure Service bus Topics  operators will be added as well

Reply:
@josh-fell @eladkal any suggestion on this above comments

Reply:
I agree with @eladkal. It would be nice if these were organized under a uniform `airflow/providers/microsoft/azure/**/asb.py` or even `airflow/providers/microsoft/azure/**/service_bus.py`

Reply:
Changed the file name to` hooks/asb.py` and moved the sub-class into the same file